azure-pipelines/jobs/release_ubuntu.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - template: ../steps/build_linux.yml
         parameters:
           build_type: 'RelWithDebInfo'
-          cmake_flags: '-DCMAKE_DEBUG_INCLUDES_LDFLAGS=ON -DENABLE_CPPUNIT=ON -DDEBUG_COMPILE=ON -DCMAKE_INSTALL_PREFIX=$PWD/staging -DCPACK_GENERATOR="TGZ;DEB"'
+          cmake_flags: '-DCMAKE_DEBUG_INCLUDES_LDFLAGS=ON -DENABLE_CPPUNIT=ON -DDEBUG_COMPILE=ON -DCMAKE_INSTALL_PREFIX=$PWD/staging -DCPACK_GENERATOR="TGZ;DEB" -DCMAKE_CXX_FLAGS="-s"'
```

Comment:
I don't know, but I can try it out and see what difference it makes.

Reply:
I tried it with `-DCPACK_STRIP_FILES=ON` but it didn't strip the `xournalpp` binary and `lintian` still complains about it.

Reply:
Maybe it would be better to put it into the `CMakeLists.txt` file via 
```
set(CPACK_STRIP_FILES ON)
```
Otherwise if you build from source and generate the package, you always have to pass `-DCPACK_STRIP_FILES=ON` to cmake. Otherwise it fails to install (on Debian at least).